documentation/book/proc-configuring-jaeger-tracer-kafka-clients.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// assembly-name.adoc
+
+[id='proc-configuring-jaeger-tracer-kafka-clients-{context}']
+= Initializing a Jaeger tracer for Kafka clients
+
+Configure and initialize a Jaeger tracer using a set of tracing environment variables.
+
+.Procedure
+
+Perform the following steps for each client application.
+
+. Add Maven dependencies for Jaeger to the `pom.xml` file for the client application. For example:
++
+[source,xml,subs=attributes+]
+----
+<dependency>
+    <groupId>io.jaegertracing</groupId>
+    <artifactId>jaeger-client</artifactId>
+    <version>$jaegerVersion</version>
```

Comment:
I think we need a `:tracing-artifact-version: 0.1.4.redhat-00002` attribute here

Reply:
What do you think, @scholzj ? Does this variable refer to the version of Jaeger, i.e. 1.14?

Reply:
Here in Strimzi it should be 0.1.4 for `opentracing-kafka-client` and 1.0.0 for `jaeger-client`. (I can share the downstream versions offline)

Reply:
Thanks. I've created two attributes:
```
:JaegerClientVersion: 1.0.0
:OpenTracingKafkaClient: 0.1.4
```

These have been inserted into the correct procedures.